### ISSUE TITLE
DDC fails on tour of heroes

### ISSUE THREAD

**Issue by RandalSchwartz** [dart-lang/angular2#455]
Using https://github.com/angular-examples/toh-6 and 1.24 SDK, fails when I add
```
web:
  compiler:
    debug: dartdevc
```
to pubspec.yaml and a javascript browser (Chrome).

I'm told the streams don't have DDC compatibility yet, so I commented out the use of uppercase, but I still get this fail, which I don't know how to remove:

```
Error compiling dartdevc module:angular_tour_of_heroes|lib/lib__hero_detail_component.template.js

[error] Element 'Location' from SDK library 'dart:html' is implicitly hidden by 'location.dart'. (package:angular_tour_of_heroes/hero_detail_component.template.dart, line 303, col 84)
[error] Element 'Location' from SDK library 'dart:html' is implicitly hidden by 'location.dart'. (package:angular_tour_of_heroes/hero_detail_component.template.dart, line 304, col 54)

Please fix all errors before compiling (warnings are okay).

```

**Comment by kwalrath** [comment]
@chalin could you comment?

**Comment by kwalrath** [comment]
@RandalSchwartz, please see https://github.com/dart-lang/site-webdev/issues/703 for some information on toh-6.

**Comment by chalin** [comment]
Hi @RandalSchwartz - thanks for the feedback. Please post angular-examples issues over at https://github.com/dart-lang/site-webdev/issues.

As I had pointed out on Gitter a few days ago:
- The stream issue has been fixed (though the repo hasn't been updated -- I'll do that now).
- The `Location` shadowing is a known issue which has already been fixed, although the fix will not be available until at least 1.25.
